pints/_mcmc/__init__.py

```diff
@@ -32,7 +32,8 @@ def __init__(self, log_likelihood, x0, sigma0=None):
 
         # Store function
         if not isinstance(log_likelihood, pints.LogLikelihood):
-            raise ValueError('Given function must extend pints.LogLikelihood')
+            if not isinstance(log_likelihood, pints.LogPDF):
```

Comment:
Can we not have something like `if not isinstance(log_likelihood, pints.LogLikelihood) and not isinstance(log_likelihood, pints.LogPDF):` here? (If we still want to keep `LogPDF` and `LogLikelihood` as two separate things, see comment above.)

Reply:
Should LogPDF perhaps be a main pints thing? So _not_ specific to the toy model?
In that case our LogPosterior could extend it, so that this could be a single check

Reply:
If we are moving this to a main pints thing, we might want to make this very clear -- the similarity and/or the difference between the two. Actually we should make it clear either we move it or not, because they very similar in some sense, and that might confuse users.

Reply:
Yeah! This is making it more clear to me why I don't like calling the thing a LogPosterior. It's the log of a PDF, why should we care what it's meaning is in the context of a particular equation?
@ben18785 ?

Reply:
Nevermind, see above

Reply:
Is LogPDF an ok name @ben18785 and @sanmitraghosh ? Or are we losing generality (could it be a pmf?)

Reply:
Ok, will have a look at this now as part of #166